tests/test_router_bgp.rb

```diff
@@ -566,6 +566,36 @@ def test_routerbgp_default_neighbor_fib_down_accelerate
     bgp.destroy
   end
 
+  def test_routerbgp_set_get_maxas_limit
+    %w(test_default test_vrf).each do |t|
+      if t == 'test_default'
+        asnum = 55
+        vrf = 'default'
+        bgp = RouterBgp.new(asnum)
+      else
+        asnum = 99
+        vrf = 'yamllll'
+        bgp = RouterBgp.new(asnum, vrf)
+      end
+      bgp.maxas_limit = 55
+      assert_equal(55, bgp.maxas_limit,
+                   "vrf #{vrf}: bgp maxas-limit should be set to '55'")
+      bgp.maxas_limit = 50
+      assert_equal(50, bgp.maxas_limit,
+                   "vrf #{vrf}: bgp maxas_limit" \
+                   "should be set to default value of '50'")
+      bgp.destroy
+    end
+  end
+
+  def test_routerbgp_default_maxas_limit
+    asnum = 55
+    bgp = RouterBgp.new(asnum)
+    assert_equal(50, bgp.maxas_limit,
+                 "bgp maxas-limit default value should be '50'")
```

Comment:
This isn't right. For this test you should:
 1. set maxas to some non default value and check the result
 2. set maxas to default using the default api, then check the result

 e.g:
```    
  bgp.maxas_limit = 3
  assert_equal(3, bgp.maxas_limit
  bgp.maxas_limit = bgp.default_maxas_limit
  assert_equal(bgp.default_maxas_limit, bgp.maxas_limit,
```